nvr/nvr.py

```diff
@@ -321,29 +343,51 @@ def get_address_type(address):
         raise ValueError
     except ValueError:
         return 'socket'
-
+    
+def create_new_pipe_name():
+    existing_pipenames = [pipename for pipename in os.listdir('\\\\.\\pipe') ]
+    pipename = "\\\\.\\pipe\\nvr-{}".format(random.randint(0, 10000))
+    
+    for n in range(10):
+        if pipename not in existing_pipenames:
+            return pipename
+    
+    print("Could not get name for new neovim pipe." "Too many existing pipes?" "Run `nvr --serverlist` to get a list of existing pipes.")
```

Comment:
Why are there spaces between the string literals?  Is each string supposed to be printed on its own line?

Reply:
@jamessan I have addressed all your review points in the latest commit! Please review as time permits.